crates/bevy_pbr/src/material.rs

```diff
@@ -258,12 +262,19 @@ impl<M: SpecializedMaterial> SpecializedMeshPipeline for MaterialPipeline<M> {
 
 impl<M: SpecializedMaterial> FromWorld for MaterialPipeline<M> {
     fn from_world(world: &mut World) -> Self {
-        let asset_server = world.get_resource::<AssetServer>().unwrap();
-        let render_device = world.get_resource::<RenderDevice>().unwrap();
+        let asset_server = world
+            .get_resource::<AssetServer>()
```

Comment:
We should replace these with the new `world.resource::<AssetServer>()` methods to avoid the need for duplicate panic messages / improve ergonomics.

Reply:
OK, Shall I do it in this pull request? or should this issue be resolved independently and across all other crates as well?

Reply:
OK, Shall I do it in this pull request? or should this issue be resolved independently and across all other crates as well?

Reply:
Personally I feel like there's no harm in merging this PR and then going through and changing `get_resource()` to `resource()` if that is the new(?) API to use. But it's up to @cart .